atomate/vasp/firetasks/run_calc.py

```diff
@@ -147,6 +148,11 @@ def run_task(self, fw_spec):
         elif job_type == "full_opt_run":
             jobs = VaspJob.full_opt_run(vasp_cmd, auto_npar=auto_npar, ediffg=self.get("ediffg"),
                                         max_steps=5, half_kpts_first_relax=False)
+        elif job_type == "neb":
```

Comment:
I am not sure why NEB needs a specific mod. You should really leave things like half_kpts and auto_gamma alone. The user sets what he wants.